chapters/bash-advanced.Rmd

```diff
@@ -912,7 +911,7 @@ $ echo $HOME
 As with loop variables (Section \@ref(bash-tools-loops)),
 the dollar sign before the variable names tells the shell
 that we want the variable's value.
-This works just like wildcard expansion
+This works just like wildcard expansion --
```

Comment:
Please use `first---second` instead of `first -- second` (em-dash is triple '-', no spaces before or after). I can search/fix these if you want.

Reply:
Good catch.